third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -435,6 +437,14 @@ function Pause_evaluateInFrame({ frameId, expression }) {
   assert(index < frames.length);
   const frame = frames[index];
 
+  if (expression === '[...arguments]') {
+    // short-circuit hackfix: return `arguments` of given frame
+    //  see: https://linear.app/replay/issue/RUN-1061#comment-fc1c3ee4
```

Comment:
This is pretty hacky, can this have a more thorough comment describing what the problem is which is being fixed, rather than a reference to a linear issue?

Reply:
Done.